src/Components/Select/views/user-option.blade.php

```diff
@@ -4,8 +4,9 @@
     :label="$label" :option="$option" :description="$description"
 >
     <div class="flex items-center gap-x-3">
-        <img src="{{ data_get($option, 'src', $src) }}" class="object-cover w-6 h-6 rounded-full shrink-0">
-
+        @if($option && !empty($src))
```

Comment:
The `img` should render from `$option` or `$src`.
```suggestion
        @if ($option || $src)
```